internal/chainnotifier_mock.go

```diff
@@ -0,0 +1,160 @@
+package test
```

Comment:
Nit: typically `internal` is just a top-level directory containing subdirectories which are the internal packages.

Reply:
Ah, gotcha. Fixed.